config/environments/development.rb

```diff
@@ -58,8 +58,6 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
```

Comment:
Might this need to be removed from the deployment project too

Reply:
I have just checked and thankfully this does not exist in the environment in the deployment project.